### DISCUSSION TITLE
Click to go to currently running cell

### DISCUSSION THREAD

**Discussion by haakon-e** [fonsp/Pluto.jl#3027] [Questions about using Pluto] [answered]
I'm using reasonably large notebooks for research, and while I really love the reactivity for reproducibility, sometimes if I make a small change somewhere, the notebook inevitably takes a while to recompute. Because the computation sometimes can be far from linear, it's hard to know what's holding things up beyond checking each cell manually for whether its timer is running or not.

Ideally, I would be able to click in the UI which indicates which cell is running, or, dreamily, have my browser "follow" the currently running cell.
<img width="332" alt="image" src="https://github.com/user-attachments/assets/afb3258e-4577-4725-81df-6ce42ba60bf7">
```[^ I want to click on the orange cell, which is currently executing, to go to it]```

similar to: #2798

**Comment by haakon-e** [comment]
Knowing nothing about web development, I imagine a reasonably "simple" toggle would be to 
Have the animated alternating color on the left of the cell:
<img width="67" alt="image" src="https://github.com/user-attachments/assets/477be9e6-07ab-47be-b216-0a1b01258291">
- be alternating dark/light grey (as above) for cells that are "pending" to be run
- be alternating dark/light green for cells that are actually running right now.

**Reply by haakon-e** [reply]
Inspecting my browser elements, it appears the area is referred to as `pluto-trafficlight`.

I wonder if setting colors differently in `pluto-cell.queued` and `pluto-cell.running` would be needed?

https://github.com/fonsp/Pluto.jl/blob/53e1981fbc7ad09d91acd8ee37a1bd6228aa5904/frontend/editor.css#L1551-L1578

-----------
Other references
https://github.com/fonsp/Pluto.jl/blob/53e1981fbc7ad09d91acd8ee37a1bd6228aa5904/frontend/components/Cell.js#L198

https://github.com/fonsp/Pluto.jl/blob/53e1981fbc7ad09d91acd8ee37a1bd6228aa5904/frontend/components/Cell.js#L322

**Reply by haakon-e** [reply]
I see there's maybe a veeeery faint difference between "queued" and "running", but my eyes struggle when actually scrolling through a notebook:
![image](https://github.com/user-attachments/assets/d9c1eb41-6055-4af1-b48a-f1bdf9e7b5fa)

Something like this color for one of the traffic light shades would be nice:
![image](https://github.com/user-attachments/assets/4f5148c6-8b02-45c4-a877-7be63142f842)

**Reply by haakon-e** [reply]
For this alternative approach, I finally figured out I could add the following line to the top of the Pluto script:
```julia
html"""
<style>
    :root {
        --dark-normal-cell-color: #006400; /* Dark green */
    }
</style>
"""
```

which looks like this for currently running cells:
<img width="455" alt="image" src="https://github.com/user-attachments/assets/550a2c03-3c55-4937-8dd2-f6a1799e6c53">

which seems reasonable. More theming options seems to be available [here](https://github.com/fonsp/Pluto.jl/blob/53e1981fbc7ad09d91acd8ee37a1bd6228aa5904/frontend/themes/light.css#L20).

**Comment by fonsp** [comment]
Hey! You can click on the loading bar on the top of the page, but let's also implement your suggestion of clicking the status tab.

**Reply by fonsp** [reply, marked as answer]
Implemented in https://github.com/fonsp/Pluto.jl/commit/a5122a702c2cb9cf36cc716832aa06e193cfe60c
